### ISSUE TITLE
createFrom -> constructor

### ISSUE THREAD

**Issue by tkrajina** [tkrajina/typescriptify-golang-structs#8]
Move everything handled in `createFrom()` to `constructor()` (with optional params)

**Comment by sampaioletti** [comment]
I have become a fan of using the Partial generic in constructors or Static factories.

i.e.
```
static initTo(target:Partial<Person>={}){
    Object.assign(init,new Person())
    return init as Person
}
static initFrom(init:Partial<Person>={}){
    let p=new Person()
    Object.assign(p,init)
    return p
}
```
The initTo method is handy if you are doing data binding and need to keep the original object refence

Just giving ideas.

**Comment by tkrajina** [comment]
Implemented in `v0.1.0`
